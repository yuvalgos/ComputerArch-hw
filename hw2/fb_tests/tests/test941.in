r 0x7faa0
r 0x7faa4
r 0x7fa74
w 0x7fabc
r 0x7fa8c
w 0x7fab4
r 0x7fa80
r 0x7fa40
w 0x7fa30
w 0x7fabc
w 0x7fa9c
r 0x7fa98
r 0x7fa54
w 0x7fa68
w 0x7fad0
w 0x7fa3c
r 0x7fa4c
w 0x7fa9c
w 0x7fa68
w 0x7fa44
r 0x7fad0
w 0x7fa98
r 0x7fa74
r 0x7fa94
w 0x7fac0
r 0x7fa4c
r 0x7fa84
r 0x7fa64
w 0x7fa78
r 0x7fab0
w 0x7fa98
w 0x7fa68
r 0x7faa4
w 0x7fac8
r 0x7fad0
r 0x7fad0
w 0x7faa0
r 0x7fa30
r 0x7fa94
r 0x7fa7c
r 0x7fa38
