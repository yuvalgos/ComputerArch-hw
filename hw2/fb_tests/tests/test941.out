L1miss=0.829 L2miss=0.676 AccTimeAvg=135.439
